openmc/plotter.py

```diff
@@ -540,3 +602,305 @@ def _calculate_xs_elem_mat(this, types, temperature=294., cross_sections=None,
                                   xs[name][line](energy_grid))
 
     return energy_grid, data
+
+
+def calculate_mgxs(this, types, orders=None, temperature=294.,
```

Comment:
I had that thought too.  My takeaway was that this was useful for other reasons so if a user did have one of those reasons, why not allow axcess